openpype/pipeline/create/creator_plugins.py

```diff
@@ -208,10 +204,33 @@ def __init__(
         # - we may use UI inside processing this attribute should be checked
         self.headless = headless
 
-        self.apply_settings(project_settings, system_settings)
+        expect_system_settings = False
+        if is_func_signature_supported(
+            self.apply_settings, project_settings
+        ):
+            self.apply_settings(project_settings)
+        else:
+            expect_system_settings = True
+            # Backwards compatibility for system settings
+            self.apply_settings(project_settings, system_settings)
+
+        init_overriden = self.__class__.__init__ is not BaseCreator.__init__
+        if init_overriden or expect_system_settings:
+            self.log.warning((
+                "WARNING: Source - Create plugin {}."
+                " System settings argument will not be passed to"
+                " '__init__' and 'apply_settings' methods in future versions"
+                " of OpenPype. Planned version to drop the support"
+                " is 3.15.6 or 3.16.0. Please contact Ynput core team if you"
```

Comment:
The version numbers mentioned here seem low. `develop` is currently on: `3.16.5-nightly.3`?

Reply:
> The version numbers mentioned here seem low. develop is currently on: 3.16.5-nightly.3?

It's 2 patch releases or 1 minor release.

Reply:
So then it should be `3.16.7 or 3.17.0` right?

Reply:
`3.16.5` is not out yet. `3.16.5-nightly.3` is preparation for `3.16.5`.

Reply:
```suggestion
                " is 3.16.6 or 3.17.0. Please contact Ynput core team if you"
```